test/jdk/java/lang/String/CompactString/MaxSizeUTF16String.java

```diff
@@ -140,7 +140,7 @@ public void testMaxUTF8_UTF16Encode() {
 
         // Strings of size min+1...min+2, throw OOME
         // The resulting byte array would exceed implementation limits
-        for (int count = min + 1; count < max; count++) {
+        for (int count = min + 2; count < max; count++) {
```

Comment:
Is this an unrelated change?

Reply:
Oops, yes. I've been debugging something in that test and accidentally made the change in the wrong workspace.